tests/cli/test_cmds_local_run.py

```diff
@@ -1615,16 +1596,7 @@ def test_missing_volumes_skipped(mock_exists):
             service='service',
             instance='instance',
             cluster='cluster',
-            system_paasta_config=SystemPaastaConfig(
-                {
-                    'volumes': [{
-                        "hostPath": "/hostPath",
-                        "containerPath": "/containerPath",
-                        "mode": "RO",
-                    }],
-                },
-                '/etc/paasta',
-            ),
+            system_paasta_config=system_paasta_config,
```

Comment:
I think most of the other changes are fine (replacing e.g. `SystemPaastaConfig({}, '/fake/config')` with the fixture), but this test cares specifically about the contents of SystemPaastaConfig, so I think it doesn't make sense to use the fixture here.

Reply:
I reverted both changes.